priv/classes.lisp

```diff
@@ -0,0 +1,69 @@
+;
+; Exploring use of classes
+;
+
+(defclass Animal ()
+
+  (defun legs ()
+    4))
+
+
+(defclass Dog (Animal)
+
+  (defmethod bark ()
+    "woof!"))
+
+
+(defclass Human (Animal)
+
+  (defun legs ()
+    2)
+
+  (defmethod pay-taxes ()
+    "Fine..."))
+
+(let ((fido (.new Dog))
+      (jane (.new Human)))
+
+  (Animal/legs)
+  (Dog/legs)
+  (Human/legs)
+
+  (.pay-taxes jane)
+  (.bark fido))
+
```

Comment:
Oh no, maybe not - is it just the end of the class? All these parentheses are a little confusing! :P